helix-view/src/document.rs

```diff
@@ -471,7 +471,7 @@ impl Document {
             if let Some(parent) = path.parent() {
                 // TODO: display a prompt asking the user if the directories should be created
                 if !parent.exists() {
-                    bail!("can't save file, parent directory does not exist");
+                    std::fs::DirBuilder::new().recursive(true).create(parent)?;
                 }
```

Comment:
I think we should at least show a status message that new directories were created (or even better a prompt like the TODO says).

Reply:
How would we display a status message or a prompt from this function?